corehq/warehouse/dbaccessors.py

```diff
@@ -40,3 +42,24 @@ def _get_ids_by_last_modified(cls, doc_types, start_datetime, end_datetime):
         )
         for result in results:
             yield result['id']
+
+
+def get_synclog_ids_by_date(start_datetime, end_datetime):
+    from casexml.apps.phone.models import SyncLog
+
+    results = SyncLog.view(
+        "sync_logs_by_date/view",
+        startkey=[json_format_datetime(start_datetime)],
+        endkey=[json_format_datetime(end_datetime)],
+        reduce=False,
+        include_docs=False
+    )
+    for result in results:
+        yield result['id']
```

Comment:
i don't think couch has a way to make this query be exclusive of the start date the way you did for forms. is that going to be a problem on the off chance we pull the same record in two separate runs? or should we just make this check the key to make sure the date doesn't equal the start date before we yield